Sources/IO/Misc/ITKImageReader/index.js

```diff
@@ -4,6 +4,13 @@ import ITKHelper from 'vtk.js/Sources/Common/DataModel/ITKHelper';
 const { convertItkToVtkImage } = ITKHelper;
 let readImageArrayBuffer = null;
 let resultPreprocessor = (result) => result;
+let uniqueArryCounter = 0;
+
+function getUniqueArrayName() {
+  uniqueArryCounter++;
+  console.log('generate unique scalars', `Scalars_${uniqueArryCounter}`);
```

Comment:
Remove console.log.

Did you want to have the name be "Scalars_${num}" or "Scalars ${num}"? Just checking because the log doesn't match the return value.

Reply:
I started with _ but them putting a space looks better